.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I'll add the whole .vscode directory to .gitignore since it seems to have been pushed by mistake initially.